keen/explorer
# Add Atlantic time

# What does this PR do?

Add a new timezone to the drop down

I selected Caracas as it appears to be the largest city in that timezone. This is critical to allow users to make queries of the eastern US during DST.

# How should it be tested?

IDK since it is a one line change to a json object. Probably check that I spelled Caracas right...

# Related issues

#203, #153
